src/Factory/Xml/Feed/FeedCountFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\SellerCenter\Factory\Xml\Feed;
+
+use Linio\SellerCenter\Exception\InvalidXmlStructureException;
+use Linio\SellerCenter\Model\Feed\FeedCount;
+use SimpleXMLElement;
+
+class FeedCountFactory
+{
+    public static function make(SimpleXMLElement $xml): FeedCount
+    {
+        self::validateStructure($xml, 'FeedCount');
+        self::validateStructure($xml->FeedCount, 'Total');
+        self::validateStructure($xml->FeedCount, 'Queued');
+        self::validateStructure($xml->FeedCount, 'Processing');
+        self::validateStructure($xml->FeedCount, 'Finished');
+        self::validateStructure($xml->FeedCount, 'Canceled');
```

Comment:
What about to create a const array of properties?
```php
  public const FEED_COUNT = 'FeedCount';
  public const PROPERTIES = [
    'Total',
    'Queued',
    'Processing',
    'Finished',
    'Canceled',
  ];
 public static function make(SimpleXMLElement $xml): FeedCount
    {
        self::validateStructure($xml, self::FEED_COUNT);
        foreach(self::PROPERTIES as property) {
           self::validateStructure($$xml->FeedCount, property);
        }
        ...
  }
  private static function validateStructure(SimpleXMLElement $xml, string $property): void
    {
         if (!property_exists($xml, $property)) {
            throw new InvalidXmlStructureException('FeedCount', $property);
        }
    }
```

Reply:
I like this! doing the change